### PR TITLE
[Added] Pinstr app to Nostr section

### LINKED ISSUE #2432: [Feature] Add https://pinstr.app/ to Nostr section
### Feature description

Hi,

Add https://pinstr.app/ to Nostr section.

### Describe the solution

Hi,

Add https://pinstr.app/ to Nostr section

### Describe alternatives

_No response_

### Additional context

_No response_

### Are you working on this?

No

### PR DESCRIPTION
**Pull request by AdarshRawat1**
### Describe the changes you have made in this PR

Added  https://pinstr.app/  to Nostr section

### Link this PR to an issue [optional]

Fixes #2432

### Type of change

- `feat`: New feature (non-breaking change which adds functionality)

### Screenshots of the changes [optional]
image

- Logo used -
image

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

File: src/app/screens/Discover/websites.json [hunk 1]
```diff
@@ -114,6 +114,12 @@
         "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/notebin-logo.png",
         "url": "https://notebin.org/"
       },
+      {
+        "title": "Pinstr",
+        "subtitle": "Social network for curating and sharing your interests",
+        "logo": "https://user-images.githubusercontent.com/100958893/238246267-ab34f64e-396a-4574-ad20-d363e5d4805a.png",
```
Comment (fczuardi) [inline src/app/screens/Discover/websites.json:120]: Do you know how the other logos were uploaded to cdn.getalby-assets.com?

Comment (AdarshRawat1) [reply src/app/screens/Discover/websites.json:120]: No, I don't know how it is done. The repository maintainers upload the logo and provide the link.  P.s. That's how it's been for me for 3 PRs.

Comment (fczuardi) [reply src/app/screens/Discover/websites.json:120]: ok, cool, just curiosity, thanks

**Comment by AdarshRawat1** [comment]
@bitcoinuser Please review the PR at your earliest convenience.

File: src/app/screens/Discover/websites.json (hunk 1 above)
Comment (reneaaron) [inline src/app/screens/Discover/websites.json:120]:
```suggestion
        "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/pinstr.png",
```

**Review by reneaaron — APPROVED** [review]
tACK

**Pull request merged**